tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -381,6 +381,18 @@ module.exports = {
         message = result.value
       })
       return message
+    },
+    getTokensOfPublicLink: async function () {
```

Comment:
The function name is not clear. It says `getTokens...` but infact it looks like it returns boolean value.

Reply:
I think you should move this entire function out of the page object. This looks mostly like assertion so should not be in the page object.

Reply:
done